devel/golangci-lint/Portfile

```diff
@@ -22,11 +23,14 @@ maintainers         {gmail.com:herby.gillot @herbygillot} \
                     {@steenzout} \
                     openmaintainer
 
-checksums           rmd160  bd69eb2c025dd13646f97dc49d0c591dee5c7b8c \
-                    sha256  f4efcc09dde3eb81ba7e2fc4230d3e99375a4b176dd28c31cab07371cf5c07db \
-                    size    1738919
+checksums           rmd160  e2c21330fbc733e38c3908569d589434571a27ce \
+                    sha256  0ae50c50dbc7603ed27a6f559dca194cb8851f030410c5635270866e78cb3400 \
+                    size    2350726
 
-build.args          ./cmd/golangci-lint
+set build_time      [clock format [clock seconds] -format "%Y-%m-%dT%H:%M:%SZ"]
```

Comment:
Even though we have some ports doing this (in some of my own as well), I think the general preference is that we do not set build time in builds as that decreases build reproducibility: https://reproducible-builds.org/docs/timestamps/